ci/Dockerfile

```diff
@@ -37,6 +37,6 @@ RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86
     && conda config --set solver libmamba
 
 # install cuML
-ARG CUML_VER=23.12
+ARG CUML_VER=24.02
```

Comment:
please change back to 23.12 and in line below change `rapidsai-nightly` to `rapidsai`

Reply:
We needed the change to the conda channel to run ci in another pr and will just merge there:  https://github.com/NVIDIA/spark-rapids-ml/pull/531/files#diff-3e80aab0c5827bd91cc241afb47e7d83cf0c2dad29dd29c6be31d7ed46d7adecR41

Please close this PR.

Reply:
Close this PR, https://github.com/NVIDIA/spark-rapids-ml/pull/531/ will fix the failure